test/classes/initializers/bothInitCalls.future

```diff
@@ -1,5 +0,0 @@
-feature request: new initializer story
-
```

Comment:
Side remark: if by the 1.14 release time we have the "new initializer story", at least the first version of it, on master, then each .future file give the specifics of the feature being requested. That way they can be reflected in STATUS. Otherwise they are not very meaningful.